### PR TITLE
adding 'Copy text' button

### PR DESCRIPTION
**Pull request by dkoode**
Ekran Resmi 2021-08-09 20 16 32

### CODE REVIEW

File: export.html [hunk 1]
```diff
@@ -22,7 +22,10 @@ <h1 class="heading">Exported Tabs</h1>
           form="usrform"
           id="export-form"
           style="max-width: 1000px; min-width: 360px"
-        ></textarea>
+          class="text-area"
+        >
+      </textarea>
+        <a href="#" class="copy-text">Copy Text</a>
```
Comment (daemon1024) [inline export.html:28]: I wonder if we can use `<button>' tag instead of anchor, this should eliminate the need of custom css in my opinion

**Review by daemon1024 — CHANGES_REQUESTED** [review]
Hey @dkoode, thank you for the pull request 😄 

I have left a review comment. 
Also, Can you please uncommit the .DS_Store files. Also is there a particular reason the file permissions were changed from 644 to 755? I don't really think they need executable access.

Let me know if you need help/clarification with any comment.

**Comment by daemon1024** [comment]
Will Close #60

File: export.html (hunk 1 above)
Comment (dkoode) [reply export.html]: Thank you for your comment and your time! I'm a bit new to github so i'm trying to get used to it :) I did what you requested. I hope it's okay this time.

**Comment by daemon1024** [comment]
Wow that was quick. Can you share the screenshot of the entire page? The code's LGTM.

File: export.html (hunk 1 above)
Comment (daemon1024) [reply export.html]: No worries, glad that you contributed :D

**Comment by dkoode** [comment]
> Wow that was quick. Can you share the screenshot of the entire page? The code's LGTM.

Sure!
lg

**Comment by daemon1024** [comment]
Can we move the "Copy" button perhaps near to the existing "Back to Manager" button? Looks a little weird to me with it's current location.

**Comment by dkoode** [comment]
lg2

Is it okay now?

**Review by daemon1024 — APPROVED** [review]
Great Work 😸 🎉

**Review by akshgpt7 — APPROVED** [review]
LGTM, thanks for the contribution @dkoode! Feel free to pick up any other issue you find interesting.

**Pull request merged**